packages/firebase_crashlytics/lib/src/firebase_crashlytics.dart

```diff
@@ -56,6 +56,34 @@ class Crashlytics {
     }
   }
 
+  Future<void> onRuntimeException(dynamic exception, dynamic stack) async {
+    print('Runtime Exception caught by Crashlytics plugin:');
+
+    bool inDebugMode = false;
+    if (!enableInDevMode) {
+      assert(inDebugMode = true);
+    }
+
+    if (inDebugMode && !enableInDevMode) {
+      print(Trace.format(stack));
+    } else {
+      // Report error
+      final List<String> stackTraceLines =
+          Trace.format(stack).trimRight().split('\n');
+      final List<Map<String, String>> stackTraceElements =
+          getStackTraceElements(stackTraceLines);
+      final dynamic result = await channel
+          .invokeMethod<dynamic>('Crashlytics#onError', <String, dynamic>{
+        'exception': "${exception.toString()}",
+        'context': 'onRuntimeException',
```

Comment:
I am not sure what to send here, because we do not have `ErrorDescription`.

Reply:
Ok, I double checked: The native method signatures the context gets send to are:
 `- (void)recordCustomExceptionName:(NSString *)name reason:(nullable NSString *)reason frameArray:(CLS_GENERIC_NSARRAY(CLSStackFrame *) *)frameArray;` on iOS. In the end the `context` gets set on the `reason` parameter, which is `nullable`, so ✅.

On Android it looks like it does no get used at all, so ✅.

Will remove then.